[0m[[0minfo[0m] [0mRunning GraphPratice [0m
[0m[[0mdebug[0m] [0mWaiting for threads to exit or System.exit to be called.[0m
[0m[[0mdebug[0m] [0mWaiting for thread run-main-6 to terminate.[0m
[0m[[0mdebug[0m] [0m  Classpath:[0m
[0m[[0mdebug[0m] [0m	/home/jeanfelix/Documents/graph/graphpratice/target/scala-2.12/classes[0m
[0m[[0mdebug[0m] [0m	/root/.ivy2/cache/org.scala-lang/scala-library/jars/scala-library-2.12.2.jar[0m
[0m[[0mdebug[0m] [0m	/root/.ivy2/cache/org.scala-graph/graph-core_2.12/jars/graph-core_2.12-1.11.5.jar[0m
[0m[[0mdebug[0m] [0m	/root/.ivy2/cache/org.apache.poi/poi/jars/poi-3.15-beta2.jar[0m
[0m[[0mdebug[0m] [0m	/root/.ivy2/cache/commons-codec/commons-codec/jars/commons-codec-1.10.jar[0m
[0m[[0mdebug[0m] [0m	/root/.ivy2/cache/org.apache.poi/poi-ooxml/jars/poi-ooxml-3.15-beta2.jar[0m
[0m[[0mdebug[0m] [0m	/root/.ivy2/cache/org.apache.poi/poi-ooxml-schemas/jars/poi-ooxml-schemas-3.15-beta2.jar[0m
[0m[[0mdebug[0m] [0m	/root/.ivy2/cache/org.apache.xmlbeans/xmlbeans/jars/xmlbeans-2.6.0.jar[0m
[0m[[0mdebug[0m] [0m	/root/.ivy2/cache/stax/stax-api/jars/stax-api-1.0.1.jar[0m
[0m[[0mdebug[0m] [0m	/root/.ivy2/cache/com.github.virtuald/curvesapi/jars/curvesapi-1.04.jar[0m
[0m[[31merror[0m] [0m(run-main-6) java.lang.NullPointerException[0m
java.lang.NullPointerException
	at GraphPratice$.$anonfun$main$2(graphpratice.scala:25)
	at scala.runtime.java8.JFunction1$mcVI$sp.apply(JFunction1$mcVI$sp.java:12)
	at scala.collection.TraversableLike$WithFilter.$anonfun$foreach$1(TraversableLike.scala:789)
	at scala.collection.immutable.Range.foreach(Range.scala:156)
	at scala.collection.TraversableLike$WithFilter.foreach(TraversableLike.scala:788)
	at GraphPratice$.main(graphpratice.scala:20)
	at GraphPratice.main(graphpratice.scala)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at sbt.Run.invokeMain(Run.scala:67)
	at sbt.Run.run0(Run.scala:61)
	at sbt.Run.sbt$Run$$execute$1(Run.scala:51)
	at sbt.Run$$anonfun$run$1.apply$mcV$sp(Run.scala:55)
	at sbt.Run$$anonfun$run$1.apply(Run.scala:55)
	at sbt.Run$$anonfun$run$1.apply(Run.scala:55)
	at sbt.Logger$$anon$4.apply(Logger.scala:84)
	at sbt.TrapExit$App.run(TrapExit.scala:248)
	at java.lang.Thread.run(Thread.java:748)
[0m[[0mdebug[0m] [0m	Thread run-main-6 exited.[0m
[0m[[0mdebug[0m] [0mInterrupting remaining threads (should be all daemons).[0m
[0m[[0mdebug[0m] [0mSandboxed run complete..[0m
java.lang.RuntimeException: Nonzero exit code: 1
	at scala.sys.package$.error(package.scala:27)
	at sbt.Defaults$$anonfun$runTask$1$$anonfun$apply$47$$anonfun$apply$48$$anonfun$apply$49.apply(Defaults.scala:803)
	at sbt.Defaults$$anonfun$runTask$1$$anonfun$apply$47$$anonfun$apply$48$$anonfun$apply$49.apply(Defaults.scala:803)
	at scala.Option.foreach(Option.scala:236)
	at sbt.Defaults$$anonfun$runTask$1$$anonfun$apply$47$$anonfun$apply$48.apply(Defaults.scala:803)
	at sbt.Defaults$$anonfun$runTask$1$$anonfun$apply$47$$anonfun$apply$48.apply(Defaults.scala:801)
	at scala.Function1$$anonfun$compose$1.apply(Function1.scala:47)
	at sbt.$tilde$greater$$anonfun$$u2219$1.apply(TypeFunctions.scala:40)
	at sbt.std.Transform$$anon$4.work(System.scala:63)
	at sbt.Execute$$anonfun$submit$1$$anonfun$apply$1.apply(Execute.scala:228)
	at sbt.Execute$$anonfun$submit$1$$anonfun$apply$1.apply(Execute.scala:228)
	at sbt.ErrorHandling$.wideConvert(ErrorHandling.scala:17)
	at sbt.Execute.work(Execute.scala:237)
	at sbt.Execute$$anonfun$submit$1.apply(Execute.scala:228)
	at sbt.Execute$$anonfun$submit$1.apply(Execute.scala:228)
	at sbt.ConcurrentRestrictions$$anon$4$$anonfun$1.apply(ConcurrentRestrictions.scala:159)
	at sbt.CompletionService$$anon$2.call(CompletionService.scala:28)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
[0m[[31merror[0m] [0m(compile:[31mrun[0m) Nonzero exit code: 1[0m
